stac_ipyleaflet/core.py

```diff
@@ -157,44 +172,47 @@ def toggle_stac_widget_display(self, b):
             if self.stac_widget.layout.display == "none":
                 self.stac_widget.layout.display = "block"
                 self.layers_widget.layout.display = "none"
-                self.aoi_widget.layout.display = "none"
+                self.interact_widget.layout.display = "none"
                 self.buttons["layers"].value = False
-                self.buttons["draw"].value = False
+                self.buttons["interact"].value = False
                 if self.draw_control_added:
                     self.remove(self.draw_control)
                     self.draw_control_added = False
+                if self.inspect_control_added:
+                    self.remove(self.inspect_control)
+                    self.inspect_control_added = False
```

Comment:
Should we combine these? I don't think it's possible to view one without the other.

Reply:
Hmm not too sure what you mean here, this is related the pointer or square area drawer control. When one tab `Point or Area` is selected and if a control was already applied, it needs to be removed, not sure if this is related to what you are asking though